addrmgr/addrmanager_test.go

```diff
@@ -105,10 +107,35 @@ func lookupFunc(host string) ([]net.IP, error) {
 	return nil, errors.New("not implemented")
 }
 
+func newAddrManager(t *testing.T, testName string, localSubnetworkID *subnetworkid.SubnetworkID) (*AddrManager, func()) {
```

Comment:
Also, I'd change the name of newAddrManager to something that makes it's clear this is a new addrManager for tests